spec/controllers/straws_controller_spec.rb

```diff
@@ -2,63 +2,82 @@
 
 RSpec.describe StrawsController, :type => :controller do
   describe "POST /straws" do
-    it "returns a single username from the list of usernames" do
+    it "should return a single username from the list of usernames" do
```

Comment:
I've been reading from http://betterspecs.org/ ... not sure how I missed this. Anywho - I'm switching to using "will" instead of "should".